draft-ietf-tls-esni.md

```diff
@@ -275,25 +281,30 @@ or if names can be added or removed from the anonymity set during the lifetime
 of a particular ECH configuration.
 
 public_name
-: The non-empty name of the client-facing server, i.e., the entity trusted to
-update the ECH configuration. This is used to correct misconfigured clients, as
-described in {{handle-server-response}}. This value MUST NOT begin or end with
-an ASCII dot and MUST be parsable as a dot-separated sequence of LDH labels, as
-defined in {{!RFC5890, Section 2.3.1}}. Clients MUST ignore any `ECHConfig`
-structure whose `public_name` does not meet these criteria. Note that these
-criteria are incomplete; they incidentally rule out textual representations of
-IPv6 addresses (see {{!RFC3986, Section 3.2.2}}), but do not exclude IPv4
-addresses in standard dotted-decimal or other non-standard notations such as
-octal and hexadecimal (see {{RFC3986, Section 7.4}}). If `public_name` contains
-a literal IPv4 or IPv6 address, the client SHOULD ignore the `ECHConfig` to
-avoid sending a non-compliant "server_name" extension on the ClientHelloOuter
-(see {{!RFC6066, Section 3}}).
+: The non-empty DNS name of the client-facing server, i.e., the entity trusted
```

Comment:
Couple minor nitpicks, just to make this less long-winded: I think we can drop "non-empty". It's already captured by both the struct definition and "DNS name". We can probably also replace "standard dotted-decimal or other non-standard notations such as octal and hexadecimal" with "dotted-decimal or other notations", since this PR ends up citing WHATWG later anyway. (Also it seems better not to get into the "standard" vs "non-standard" rabbithole.)

Reply:
👍 I'm fine with this, too :)